examples/cli/readme.md

```diff
@@ -1 +1,619 @@
-placeholder
\ No newline at end of file
+# Learning the `benchit` CLI
+
+This document is a tutorial for exploring the different features of the `benchit` command line interface (CLI). You can learn the details of those features in the [Tools User Guide](https://github.com/groq/mlagility/blob/main/docs/tools_user_guide.md) and learn about their implementation in the [Code Organization Guide](https://github.com/groq/mlagility/blob/main/docs/code.md).
+
+We've created this tutorial document because `benchit` is a CLI that benchmarks the contents of `.py` scripts. So all of the `.py` scripts in the `examples/cli/scripts` directory are meant to be fed into `benchit` to demonstrate some specific functionality.
+
+Once you've familiarized yourself with these features, head over to the [`models` directory](https://github.com/groq/mlagility/tree/main/models) to learn how to use `benchit` with real world machine learning models.
+
+# Table of Contents
+
+TODO
+
+# Tutorials
+
+All of the tutorials assume that your command line is in the same location as this readme file (`examples/cli`).
+
+## Hello World
+
+We can perform a basic invocation of `benchit` to benchmark a PyTorch model by simply running the following command:
+
+```
+benchit scripts/hello_world.py
+```
+
+That commands `benchit` benchmark `hello_world.py` on your CPU. Specifically, `benchit` takes the following actions:
+1. Pass `scripts/hello_world.py` as the `input_script` to the `benchmark` command of `benchit`.
+  - _Note_: `benchit <.py file>` is a shorthand for `benchit benchmark <.py file>`.
+1. Run `hello_world.py` against a profiler and look for machine learning models.
```

Comment:
"pytorch/tensorflow models" instead of "machine learning models"?

Reply:
It's a debate :( 

If we say "pytorch/keras" we are being precise, but when we add onnx or hummingbird or whatever we have to go back and edit every reference. Also "pytorch/keras/onnx/hummingbird/jax/matlab" is a mouthful. OTOH, "machine learning models" is less precise, but more maintainable. Thoughts?

Reply:
Maybe "models from supported machine learning frameworks (e.g. Pytorch)" is a good compromise here?

Reply:
I like that, thanks!